Mathlib/CategoryTheory/Limits/Shapes/MultiequalizerPullback.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Limits.Shapes.Multiequalizer
+import Mathlib.CategoryTheory.Limits.Shapes.Pullback.CommSq
+
+/-!
+# Multicoequalizers that are pushouts
+
+In this file, we show that a multicoequalizer for
+for `I : MultispanIndex (.ofLinearOrder ι) C` is also
+a pushout when `ι` has exactly two elements.
+
+-/
+
+namespace CategoryTheory.Limits.Multicofork.IsColimit
+
+variable {C : Type*} [Category C] {J : MultispanShape} [Unique J.L]
+  {I : MultispanIndex J C} (c : Multicofork I)
+  (h : {J.fst default, J.snd default} = Set.univ) (h' : J.fst default ≠ J.snd default)
+
+namespace isPushout
```

Comment:
Shouldn’t `multicofork` be in the `CategoryTheory.Limits.PushoutCocone` namespace so that we can use dot notation?

Reply:
Your, call, but even as an "implementation" definition, I feel that `CategoryTheory.Limits.Multicofork.IsColimit.isPushout.multicofork` is one hell of a name.